tensorboard/summary_test.py

```diff
@@ -37,14 +37,6 @@
     'text',
 ])
 
-# The subset of `STANDARD_PLUGINS` for which we do not currently have
-# functions to generate a summary protobuf outside of a TensorFlow
-# graph. This set should ideally be empty; any entries here should be
-# considered temporary.
-PLUGINS_WITHOUT_PB_FUNCTIONS = frozenset([
-    'pr_curve',  # TODO(@chihuahua, #445): Fix this.
```

Comment:
Up to you whether to delete this set, changing the code below, or simply make it empty and leave the check in place. Either is fine with me.

Reply:
Hmm ... I suppose if we now have a chance to pursue the ideal, how about we remove this structure, so that anyone who wishes to make an exception would have to reintroduce a similar structure ... more impetus for implementing a `pb` method.

Reply:
Makes sense to me!